share/spack/gitlab/cloud_pipelines/stacks/e4s-oneapi/spack.yaml

```diff
@@ -31,10 +32,7 @@ spack:
     openblas:
       variants: threads=openmp
     trilinos:
-      variants: +amesos +amesos2 +anasazi +aztec +belos +boost +epetra +epetraext
-        +ifpack +ifpack2 +intrepid +intrepid2 +isorropia +kokkos +ml +minitensor +muelu
-        +nox +piro +phalanx +rol +rythmos +sacado +stk +shards +shylu +stokhos +stratimikos
-        +teko +tempus +tpetra +trilinoscouplings +zoltan +zoltan2 +superlu-dist gotype=long_long
+      variants: +amesos +amesos2 +anasazi +aztec +belos +boost +epetra +epetraext +ifpack +ifpack2 +intrepid +intrepid2 +isorropia +kokkos +ml +minitensor +muelu +nox +piro +phalanx +rol +rythmos +sacado +stk +shards +shylu +stokhos +stratimikos +teko +tempus +tpetra +trilinoscouplings +zoltan +zoltan2 +superlu-dist gotype=long_long
```

Comment:
undo

Reply:
Will do, but I think this one is from your suggestion that the best linter is a round-trip on load/dump :stuck_out_tongue:

Reply:
Meaning if you do:
```
spack -e ... add foo
spack -e ... remove foo
```
you'll get the variants written as they are above.

Reply:
Sure, but this is not about style